### PR TITLE
feat: add logging management

### PR DESCRIPTION
**Pull request by Thomgrus**
## Type of modification

* [ ] Breaking change
* [x] New Feature
* [ ] Bug Fix
* [ ] Chore (refactor, documentation, tests... all the changes with no impact on ARA functionalities.) 

## Changes description

Add log management

## Technical description

confère doc/tech/LOGGING.adoc

## PR CheckList

Please make sure your PullRequest respect all those items :

* [x] Your PR's title has the prefix : `feat:`, `fix:` or `chore:`
* [x] You have asked a review from one of the ARA maintainer in your PR.
* [x] If your PR is related to an issue, add the issue's number in it.
* [x] All the code you added is documented.
* [x] All the code you added is tested and the  tests are in success.
* [x] You already signed the [Contributor License Agreement](https://github.com/Decathlon/ara/blob/main/doc/contributing/contributor-licence-agreement.adoc) and give us the document

### CODE REVIEW

File: doc/tech/LOGGING.adoc [hunk 1]
```diff
@@ -0,0 +1,9 @@
+= Logging
+
+== API
+
+You can use different logging manage by spring profiles:
```
Comment (omar-chahbouni-decathlon) [inline doc/tech/LOGGING.adoc:5]: You can use many logging management systems using different Spring profiles:

File: doc/tech/LOGGING.adoc [hunk 2]
```diff
@@ -0,0 +1,9 @@
...
+
+* console with `logging-console` profile
```
Comment (omar-chahbouni-decathlon) [inline doc/tech/LOGGING.adoc:7]: no need to add profile at end of your sentence 
* console with `logging-console`

File: doc/tech/LOGGING.adoc [hunk 3]
```diff
@@ -0,0 +1,9 @@
...
+* json with `logging-json` profile
```
Comment (omar-chahbouni-decathlon) [inline doc/tech/LOGGING.adoc:8]: idem

File: doc/tech/LOGGING.adoc [hunk 4]
```diff
@@ -0,0 +1,9 @@
...
+* gcp stackdriver with `logging-api` profile
```
Comment (omar-chahbouni-decathlon) [inline doc/tech/LOGGING.adoc:9]: idem

**Review by omar-chahbouni-decathlon — CHANGES_REQUESTED** [review]
Just some sentence to change and you're good to go ;)

**Review by TroyonGuillaume — APPROVED** [review]

**Review by omar-chahbouni-decathlon — APPROVED** [review]

**Pull request merged**